### DISCUSSION TITLE
Issues with Okta OIDC Integration

### DISCUSSION THREAD

**Discussion by cleight** [netbox-community/netbox#14574] [Help Wanted!]
Looking for some assistance with Okta integration. I'm currently getting the following error when trying to auth to Okta:
<img width="599" alt="image" src="https://github.com/netbox-community/netbox/assets/23556848/07b631a1-c18b-41b9-a08d-cbd4935a9fad">

Looks like everything is configured correctly on the Okta side, and feel like it is something on the NetBox side. I just can't pinpoint the issue...

Okta Settings:
<img width="366" alt="image" src="https://github.com/netbox-community/netbox/assets/23556848/f12c7eee-1367-4737-8a0b-f4bcbc13f7ff">
<img width="363" alt="image" src="https://github.com/netbox-community/netbox/assets/23556848/429e723c-2127-4287-aaf7-972a9f6e7c0c">

Configuration.py settings:
<img width="643" alt="image" src="https://github.com/netbox-community/netbox/assets/23556848/c906de80-19f9-406e-a279-3bdf4d6708b7">

**Comment by cleight** [comment]
Anyone have any suggestions on this error?

**Reply by mshrud002** [reply]
@cleight , did you get a solution to the above problem, i'm experiencing same issue?
